2.6.md

```diff
@@ -1034,14 +1034,19 @@ The presence of `Banner` (Section 3.2.6), `Video` (Section 3.2.7), and/or `Nativ
     <td>Details about ad slots being refreshed automatically. (Section 3.2.33)</td>
     </td>
   </tr>
+  <tr>
+    <td><code>channelPkg</code></td>
```

Comment:
Changes to the actual spec need to go through the Working Group. Please contact @hillslatt @wittjill for details.